plugins/easy-rollback.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: easy-rollback
+spec:
+  version: v1.0.19
+  homepage: https://github.com/Trendyol/kubectl-easy-rollback
+  shortDescription: Show previously deployed image versions and easily rollback to selected one.
```

Comment:
this is too long, it will be truncated.
choose a shorter description.

Reply:
Done 👍